snapcraft.yaml

```diff
@@ -53,6 +53,8 @@ apps:
     completer: kubectl.bash
   add-node:
     command: microk8s-add-node.wrapper
+  add-token:
```

Comment:
I am not much in favor of exposing the tokens mechanism to the end user. I believe it is something internal and if there is a real need lets keep it internal so we can change it in the future if we need to.

Reply:
One of the idea of being able to specify a token by a user, is that it is difficult to extract the token automatically if it is generated.
This is also a feature which kubeadm also uses to allow it to be easily used by automation tools such as terraform or ansible.

Reply:
Can we allow `add-node` to take in a token args? We can then hide the `add-token` for public use.

Reply:
Make add-token wrapper private.  Still maintain the usual way `microk8s.add-node`, will generate one time token use.